### PR TITLE
silenced openssl x509 warning using -in

### PR DESCRIPTION
**Pull request by zeckma-tech**
Using make-ca, I had the warning for each cert described in issue #29 
I tried using -in but still having it be piped to the openssl x509 command to no avail, so I put the result of the printf(...) to awkinputfile.awktmp so it can be used as an input file and thus silences the warning. 
Does this work at all, both in terms of functionality and how any of you maintainers feel? 

Tested the commit with OpenSSL 3.2.0, compiled using the LFS instructions - no warning regarding stdin, -in, and -new for me.

### CODE REVIEW

**Comment by xry111** [comment]
Per https://github.com/openssl/openssl/issues/22893 we can use `-in /dev/stdin` to silence the warning w/o introducing temporary file.

**Review by xry111 — CHANGES_REQUESTED** [review]
.

File: help2man
Comment (xry111) [inline help2man:1]: Please remove this permission change.  Even if we intend to change it, the change should still be in a different PR.

Comment (zeckma-tech) [reply help2man:1]: Sorry, forgot to run "make clean" before committing and pushing.

**Review by xry111 — APPROVED** [review]
LGTM.

@pierre-labastie Ok for a squash-and-merge and a new release?

**Comment by xry111** [comment]
I plan to merge this if there's no objection in a week.

**Comment by djlucas** [comment]
The proposed fix looks good (assuming it works, have not tested). I wanted to comment on the first proposed fix and note that it revealed another error that probably needs to be addressed (statically named tempfile).

**Comment by zeckma-tech** [comment]
Should I add another commit to this PR addressing the statically named tempfile?

**Pull request merged**